stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  required: true
+  default: "5Gi"
+  description: "Trivy Persistent Volume Size"
+  type: string
+  label: Trivy Volume Size
+  group: "Trivy Settings"
+  show_if: "trivy.enabled=true&&persistence.enabled=true"
+- variable: persistence.persistentVolumeClaim.trivy.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Trivy
+  group: "Trivy Settings"
+  show_if: "trivy.enabled=true&&persistence.enabled=true"
+- variable: trivy.gitHubToken
+  default: ""
+  description: "A GitHub access token used to download the Trivy DB without incurring rate limiting"
+  type: string
+  label: Set a GitHub access token for Trivy
+  show_if: "trivy.enabled=true"
+  group: "Trivy Settings"
+# notary options
+- variable: notary.enabled
+  default: true
+  description: "Enable Notary for publishing and managing trusted collections of content like Docker images"
+  type: boolean
+  label: Enable Notary
+  group: "Notary Settings"
+- variable: core.replicas
+  default: 1
+  description: "Number of core service pods"
+  type: int
+  label: Core Service Replica Count
+  group: "Scale and High Availability Settings"
+- variable: portal.replicas
+  default: 1
+  description: "Number of portal service pods"
+  type: int
+  label: Portal Service Replica Count
+  group: "Scale and High Availability Settings"
+- variable: nginx.replicas
+  default: 1
+  description: "Number of nginx service pods"
+  type: int
+  label: Nginx Service Replica Count
+  show_if: "expose.type!=ingress"
+  group: "Scale and High Availability Settings"
+- variable: registry.replicas
+  default: 1
+  description: "Number of registry service pods"
+  type: int
+  label: Registry Service Replica Count
+  group: "Scale and High Availability Settings"
+- variable: jobservice.replicas
+  default: 1
+  description: "Number of jobservice service pods"
+  type: int
+  label: JobService Replica Count
+  group: "Scale and High Availability Settings"
+- variable: trivy.replicas
+  default: 1
+  description: "Number of trivy service pods"
+  type: int
+  label: Trivy Service Replica Count
+  show_if: "trivy.enabled=true"
+  group: "Scale and High Availability Settings"
+- variable: notary.server.replicas
+  default: 1
+  description: "Number of notary server service pods"
+  type: int
+  label: Notary Server Service Replica Count
+  show_if: "notary.enabled=true"
+  group: "Scale and High Availability Settings"
+- variable: notary.signer.replicas
+  default: 1
+  description: "Number of notary signer service pods"
+  type: int
+  label: Notary Signer Service Replica Count
+  show_if: "notary.enabled=true"
+  group: "Scale and High Availability Settings"
+- variable: updateStrategy.type
+  default: "RollingUpdate"
+  description: "The update strategy for deployments with persistent volumes. Set it to 'Recreate' when 'RWM' for volumes isn't supported"
```

Comment:
_RWM access mode_, not just RWM.
Or even ReadWriteMany, that's actually the correct value.

That reminds me: shouldn't you cover `accessMode` variables as well?

Reply:
done. And I also added an access mode field for registry and jobservice (only those make sense).